pyvista/core/utilities/geometric_objects.py

```diff
@@ -50,6 +50,9 @@ def Cylinder(
 ):
     """Create the surface of a cylinder.
 
+    Cylinder function rotates the CylinderSource's PolyData in its own way.
+    Therefore the plot of CylinderSource object and Cylinder function are different.
+
```

Comment:
1. The content of the document is classified as a warning, so please refer to the following section and add it as a warning
https://github.com/pyvista/pyvista/blob/0baf8ca68c1c6109769c78e237865a083cfeeca5/pyvista/core/filters/data_set.py#L3339-L3343

1. Please indicate specifically how the coordinates are being changed in value. If it depends on a parameter, please add that information as well.

1. Please add the same information to the `CylinderSource` class so that users can realize it when they read each document.

Reply:
> Please add the same information to the `CylinderSource` class so that users can realize it when they read each document.
Would it be the same warning in both the cases?

Reply:
Yes, please.